ui/lib/kv/addon/components/kv-version-dropdown.hbs

```diff
@@ -10,6 +10,7 @@
         {{#each @metadata.sortedVersions as |versionData|}}
           <li data-test-version={{versionData.version}} class="action">
             {{#if @onSelect}}
+              {{! TODO Hds::Button manual update }}
```

Comment:
Given that there are multiple different states with different icons that have varying colors I thought best to leave this as is for now.

Reply:
Good call - this is probably a good case to eventually use the Hds dropdown component